lib/PrimitiveHelpers.js

```diff
@@ -163,6 +186,14 @@ function transformPrimitives(gltf, primitives, transform) {
     }
 }
 
+/**
+ * Get a hash table like object with the material as the hash and the
+ * primitives that use the material in an array as the bucket.
```

Comment:
This description can be simplified to `Gets primitives sorted by material.` The return can describe what the object looks like.